package.json

```diff
@@ -159,11 +161,11 @@
         {
           "assets": [
             {
-              "path": "dist/webex-components.es.min.js",
+              "path": "dist/es/webex-components.es.min.js",
```

Comment:
What's the purpose of having both `es` in the folder name and file name?

Reply:
I added the folders because it makes the `dist` folder much cleaner.
The bundle type was already part of the name. I left it because I think it's useful when locally copying the bundles for testing purposes, as it's easier to tell what the contents are when the bundle file is moved outside the `dist` folder context.